modules/core/src/main/java/org/apache/ignite/internal/processors/authentication/UserAuthenticateRequestMessage.java

```diff
@@ -17,26 +17,27 @@
 
 package org.apache.ignite.internal.processors.authentication;
 
-import java.nio.ByteBuffer;
+import org.apache.ignite.internal.Order;
 import org.apache.ignite.internal.util.typedef.internal.S;
 import org.apache.ignite.lang.IgniteUuid;
 import org.apache.ignite.plugin.extensions.communication.Message;
-import org.apache.ignite.plugin.extensions.communication.MessageReader;
-import org.apache.ignite.plugin.extensions.communication.MessageWriter;
 
 /**
  * Message is sent from client to coordinator node when a user needs to authorize on client node.
  */
 public class UserAuthenticateRequestMessage implements Message {
+    /** Request ID. */
+    @Order(0)
+    private IgniteUuid id = IgniteUuid.randomUuid();
```

Comment:
Please keep order of fields declaration to reduce amount of changes. Order of fields in serialized form doesn't matter. Then we can assign Order(2) to the field